packages/gasket-plugin-docs/lib/docs-setup.js

```diff
@@ -19,7 +20,8 @@ module.exports = function docsSetup() {
     transforms: [
       txGasketPackageLinks,
       txGasketUrlLinks,
-      txAbsoluteLinks
+      txAbsoluteLinks,
+      txLicenseLinks
```

Comment:
If you just include the LICENSE.md in the files is this extra transform necessary?

Reply:
I believe so but I'll doublecheck. All the links to the License file are broken either with the file present.